lib/Data/Database.php

```diff
@@ -768,9 +860,11 @@ private static function _createCommentTable()
     private static function _createConfigTable()
     {
         list($main_key, $after_key) = self::_getPrimaryKeyClauses('id');
+        $charType                   = self::$_type === 'oci' ? 'VARCHAR2(16)' : 'CHAR(16)';
+        $textType                   = self::$_type === 'oci' ? 'VARCHAR2(4000)' : 'TEXT';
```

Comment:
Please use the function you have created for that purpose/DB switch here.

Reply:
Well spotted and duplicate as well, addressed in 35ef64f